R/survival-survreg-tidiers.R

```diff
@@ -2,7 +2,8 @@
 #' @template title_desc_tidy
 #'
 #' @param x An `survreg` object returned from [survival::survreg()].
-#' @param conf.level confidence level for CI
+#' @param conf.int Logical indicating whether or not to include a confidence interval in the tidied output. Defaults to FALSE.
```

Comment:
For consistency throughout the documentation it would nice to use `@template param_confint` here, which will pull in the boilerplate from `man-roxygen/param_confint.R` about how the `conf.int` and `conf.level` arguments are used.

Reply:
Will do, and thanks, I learned something new today! 😁